src/store/PopularCategories/slice.ts

```diff
@@ -0,0 +1,39 @@
+import { LOAD_STATUSES } from "src/types/loadStatuses";
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import type { PopularCategories } from "src/types/general";
+import { api } from "src/api/api";
+
+export interface State {
+  loadStatus: LOAD_STATUSES;
+  popularItems: PopularCategories[];
+}
+
+const initialState: State = {
+  loadStatus: LOAD_STATUSES.UNKNOWN,
+  popularItems: [],
+};
+
+const SLICE_NAME = "popular_categories";
```

Comment:
лучше называть слайс как и папку, тогда точно не будет 2 одинаков имени (файловая система не даст создать), которые будут перезатирать друг друга
```suggestion
const SLICE_NAME = "popularCategories";
```